pom.xml

```diff
@@ -1105,7 +1105,7 @@
             <dependency>
                 <groupId>com.github.ben-manes.caffeine</groupId>
                 <artifactId>caffeine</artifactId>
```

Comment:
Where this is used? Can we drop it maybe?

Reply:
It is used. We can rewrite call sites